aws/redshift/tables/student_activity_csp_csd.sql

```diff
@@ -1,10 +1,11 @@
-drop table if exists analysis.student_activity;
-CREATE table analysis.student_activity AS
+drop table if exists analysis.student_activity_csp_csd;
+CREATE table analysis.student_activity_csp_csd AS
   SELECT u_teachers.studio_person_id,
+         school_year,
          COUNT(DISTINCT se.id) sections,
          COUNT(DISTINCT us.user_id) students,
          COUNT(DISTINCT CASE WHEN u_students.gender = 'f' THEN us.user_id ELSE NULL END) students_female,
-         COUNT(DISTINCT CASE WHEN u_students.gender IN ('m','f','n') THEN us.user_id ELSE NULL END) students_gender,
+         COUNT(DISTINCT CASE WHEN u_students.gender IN ('m','f') THEN us.user_id ELSE NULL END) students_gender,
```

Comment:
Maddie added this change to include folks who responded as non-binary to the denominator of the gender calculation we do here. I think we need to do a more holistic review of how to best include non-binary folks in gender calculations, but I think leaving her change here is good to do for now.